### ISSUE TITLE
Upload preset not found

### ISSUE THREAD

**Issue by itsAdelS** [adrianhajdin/imaginify#11]
Hi Everyone, im running into this error when trying to upload any image. I have checked all of my code against adrians and even copied his pages to the T I cannot figure out where to look to find this error and correct it.
![Screenshot 2024-03-02 122402](https://github.com/adrianhajdin/ai_saas_app/assets/115944408/fd8b86aa-779a-46e2-91f0-70532175a4cd)

**Comment by itsAdelS** [comment]
I have found where there error is from the console but dont know where to find the error in the code if there is one because everything compiles smoothly aside from some hydration errors.

Warning: Extra attributes from the server: data-dashlane-rid,data-form-type,data-kwimpalastatus,data-kwimpalaid
    at input
    at _c (webpack-internal:///(app-pages-browser)/./components/ui/input.tsx:13:11)
    at eval (webpack-internal:///(app-pages-browser)/./node_modules/@radix-ui/react-slot/dist/index.mjs:42:23)
    at eval (webpack-internal:///(app-pages-browser)/./node_modules/@radix-ui/react-slot/dist/index.mjs:16:23)
    at eval (webpack-internal:///(app-pages-browser)/./components/ui/form.tsx:124:14)
    at div
    at eval (webpack-internal:///(app-pages-browser)/./components/ui/form.tsx:74:11)
    at Controller (webpack-internal:///(app-pages-browser)/./node_modules/react-hook-form/dist/index.esm.mjs:539:18)
    at FormField (webpack-internal:///(app-pages-browser)/./components/ui/form.tsx:29:14)
    at CustomField (webpack-internal:///(app-pages-browser)/./components/shared/CustomField.tsx:13:11)
    at form
    at FormProvider (webpack-internal:///(app-pages-browser)/./node_modules/react-hook-form/dist/index.esm.mjs:151:13)
    at TransformationForm (webpack-internal:///(app-pages-browser)/./components/shared/TransformationForm.tsx:53:11)
    at section
    at InnerLayoutRouter (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:242:11)
    at RedirectErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:73:9)
    at RedirectBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:81:11)
    at NotFoundBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/not-found-boundary.js:84:11)
    at LoadingBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:335:11)
    at ErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/error-boundary.js:161:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:152:9)
    at ScrollAndFocusHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:227:11)
    at RenderFromTemplateContext (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/render-from-template-context.js:16:44)
    at OuterLayoutRouter (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:354:11)
    at InnerLayoutRouter (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:242:11)
    at RedirectErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:73:9)
    at RedirectBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:81:11)
    at NotFoundBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/not-found-boundary.js:84:11)
    at LoadingBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:335:11)
    at ErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/error-boundary.js:161:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:152:9)
    at ScrollAndFocusHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:227:11)
    at RenderFromTemplateContext (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/render-from-template-context.js:16:44)
    at OuterLayoutRouter (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:354:11)
    at InnerLayoutRouter (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:242:11)
    at RedirectErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:73:9)
    at RedirectBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/redirect-boundary.js:81:11)
    at NotFoundBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/not-found-boundary.js:84:11)
    at LoadingBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:335:11)
    at ErrorBoundary (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/error-boundary.js:161:11)
    at InnerScrollAndFocusHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/components/layout-router.js:152:9)
    at ScrollAndFocusHandler (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/
window.console.error @ app-index.js:33
console.error @ hydration-error-info.js:45
printWarning @ react-dom.development.js:94
error @ react-dom.development.js:68
warnForExtraAttributes @ react-dom.development.js:32589
diffHydratedProperties @ react-dom.development.js:34970
hydrateInstance @ react-dom.development.js:35973
prepareToHydrateHostInstance @ react-dom.development.js:7299
completeWork @ react-dom.development.js:19712
completeUnitOfWork @ react-dom.development.js:25843
performUnitOfWork @ react-dom.development.js:25648
workLoopConcurrent @ react-dom.development.js:25623
renderRootConcurrent @ react-dom.development.js:25579
performConcurrentWorkOnRoot @ react-dom.development.js:24432
workLoop @ scheduler.development.js:256
flushWork @ scheduler.development.js:225
performWorkUntilDeadline @ scheduler.development.js:534
app-index.js:33 Warning: An error occurred during hydration. The server HTML was replaced with client content in <#document>.
window.console.error @ app-index.js:33
console.error @ hydration-error-info.js:45
printWarning @ react-dom.development.js:94
error @ react-dom.development.js:68
errorHydratingContainer @ react-dom.development.js:36135
recoverFromConcurrentError @ react-dom.development.js:24521
performConcurrentWorkOnRoot @ react-dom.development.js:24470
workLoop @ scheduler.development.js:256
flushWork @ scheduler.development.js:225
performWorkUntilDeadline @ scheduler.development.js:534
10on-recoverable-error.js:21 Uncaught Error: Hydration failed because the initial UI does not match what was rendered on the server.
    at throwOnHydrationMismatch (react-dom.development.js:7088:9)
    at popHydrationState (react-dom.development.js:7437:9)
    at completeWork (react-dom.development.js:19707:31)
    at completeUnitOfWork (react-dom.development.js:25843:14)
    at performUnitOfWork (react-dom.development.js:25648:5)
    at workLoopConcurrent (react-dom.development.js:25623:5)
    at renderRootConcurrent (react-dom.development.js:25579:9)
    at performConcurrentWorkOnRoot (react-dom.development.js:24432:38)
    at workLoop (scheduler.development.js:256:34)
    at flushWork (scheduler.development.js:225:14)
    at MessagePort.performWorkUntilDeadline (scheduler.development.js:534:21)
throwOnHydrationMismatch @ react-dom.development.js:7088
popHydrationState @ react-dom.development.js:7437
completeWork @ react-dom.development.js:19707
completeUnitOfWork @ react-dom.development.js:25843
performUnitOfWork @ react-dom.development.js:25648
workLoopConcurrent @ react-dom.development.js:25623
renderRootConcurrent @ react-dom.development.js:25579
performConcurrentWorkOnRoot @ react-dom.development.js:24432
workLoop @ scheduler.development.js:256
flushWork @ scheduler.development.js:225
performWorkUntilDeadline @ scheduler.development.js:534
react-dom.development.js:16472 Uncaught Error: There was an error while hydrating. Because the error happened outside of a Suspense boundary, the entire root will switch to client rendering.
    at updateHostRoot (react-dom.development.js:16472:57)
    at beginWork$1 (react-dom.development.js:18410:14)
    at beginWork (react-dom.development.js:26791:14)
    at performUnitOfWork (react-dom.development.js:25637:12)
    at workLoopSync (react-dom.development.js:25353:5)
    at renderRootSync (react-dom.development.js:25308:7)
    at recoverFromConcurrentError (react-dom.development.js:24525:20)
    at performConcurrentWorkOnRoot (react-dom.development.js:24470:26)
    at workLoop (scheduler.development.js:256:34)
    at flushWork (scheduler.development.js:225:14)
    at MessagePort.performWorkUntilDeadline (scheduler.development.js:534:21)
updateHostRoot @ react-dom.development.js:16472
beginWork$1 @ react-dom.development.js:18410
beginWork @ react-dom.development.js:26791
performUnitOfWork @ react-dom.development.js:25637
workLoopSync @ react-dom.development.js:25353
renderRootSync @ react-dom.development.js:25308
recoverFromConcurrentError @ react-dom.development.js:24525
performConcurrentWorkOnRoot @ react-dom.development.js:24470
workLoop @ scheduler.development.js:256
flushWork @ scheduler.development.js:225
performWorkUntilDeadline @ scheduler.development.js:534

**Comment by itsAdelS** [comment]
TLDR i fixed the upload preset issue i was having now I have to focus on the transformed image portion. once transformed it looks like a broken image place holder and now I have to figure out where that error is

**Comment by itsAdelS** [comment]
update! I HAVE FIXED IT booya all on my own lets go babyyyyyy

**Comment by MusaMasroor** [comment]
@itsAdelS how did you fixed it ?

**Comment by itsAdelS** [comment]
First I made sure that cloudiness folder name was correct, I had a capitalization issue, that got the upload to pass but the transformation wasn’t working. Then I noticed one of the secureURL’s was still secureUrl so I fixed that. Now I’m good!

**Comment by XaviDigi** [comment]
Hey do you mind sharing your code? I have been unable to make it work, its been 2 days now.. I can login, pay but i cant see the form in any links.
